composer.json

```diff
@@ -16,11 +16,11 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/framework-bundle" : "~2.3",
-        "fsi/admin-bundle" : "1.1.x-dev"
+        "symfony/framework-bundle" : "^2.3",
+        "fsi/admin-bundle" : "~1.1@dev"
```

Comment:
we can't change dependnecies to ``~1.1`` because there is not ``1.1.0`` tag, thats why it require ``@dev``.
We can leave admin-bundle dependency as it is right now and after release of admin-bundle 2.0.0 change it but it will means that all versions 1.0.0 of other bundles points to version 2.0.0 of admin-bundle which is bc breake because before release of 2.0.0 they were pointing to 1.1.*@dev

I'm not sure whats best here to be honest. Maybe downgrading those dependencies to admin-bundle: ^1.0 and creating version 2.0.0 of each bundle after admin-bundle release?

And about those BC Breakes, theye are there anyway, not tagging version 1.1 will not change that fact.